src/main/java/kr/codesqaud/cafe/Repository/user/JdbcUserRepository.java

```diff
@@ -0,0 +1,64 @@
+package kr.codesqaud.cafe.Repository.user;
+
+import kr.codesqaud.cafe.domain.Article;
+import kr.codesqaud.cafe.domain.User;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.core.RowMapper;
+import org.springframework.jdbc.core.namedparam.MapSqlParameterSource;
+import org.springframework.jdbc.core.simple.SimpleJdbcInsert;
+import org.springframework.stereotype.Repository;
+
+import javax.sql.DataSource;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.concurrent.ConcurrentHashMap;
+
+@Repository
+public class JdbcUserRepository implements UserRepository{
+
+    private final JdbcTemplate jdbcTemplate;
+    private final SimpleJdbcInsert simpleJdbcInsert;
+
+    public JdbcUserRepository(DataSource dataSource) {
+        this.jdbcTemplate = new JdbcTemplate(dataSource);
+        this.simpleJdbcInsert = new SimpleJdbcInsert(dataSource)
+                .withTableName("member")
+                .usingGeneratedKeyColumns("id");
+    }
+
+    @Override
+    public void save(final User user) {
+        Map<String, Object> parameters = new ConcurrentHashMap<>();
+        parameters.put("id", user.getId());
+        parameters.put("password", user.getPassword());
+        parameters.put("name", user.getName());
+        parameters.put("email",user.getEmail());
+
+        Number key = simpleJdbcInsert.executeAndReturnKey(new MapSqlParameterSource(parameters));
+        user.setId(key.toString());
+
+    }
+
+    @Override
+    public Optional<User> findById(final String id) {
+        String sql = "select * from member where id = ?";
+        return jdbcTemplate.query(sql, userRowMapper, id).stream().findAny();
+    }
+
+    @Override
+    public Optional<User> findByEmail(final String email) {
+        String sql = "select * from member where email = ?";
+        return jdbcTemplate.query(sql, userRowMapper, email).stream().findAny();
+    }
+
+    @Override
+    public List<User> findAllUser() {
+        String sql = "SELECT id, password, name, email  FROM member";
+        return jdbcTemplate.query(sql, userRowMapper);
+    }
+
+    private RowMapper<User> userRowMapper =  (rs, rowNum) ->
+            new User(rs.getString("id"),rs.getString("password"),
+                    rs.getString("name"),rs.getString("email"));
```

Comment:
```suggestion
            new User(
                 rs.getString("id"),
                 rs.getString("password"),
                 rs.getString("name"),
                 rs.getString("email")
             );
```

이렇게 써주는게 가독성이 더 좋겠죠?

Reply:
해당 부분은 생각치 못했는데 
말씀하신대로 세로로 쭉 나열하는 것이 가독성이 더 좋은 것 같네요.
수정하도록 하겠습니다.